modules/vaos/app/schemas/appointments_index.json

```diff
@@ -0,0 +1,362 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "type": "object",
+  "required": [
+    "data"
+  ],
+  "properties": {
+    "data": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "id",
+          "identifier",
+          "kind",
+          "location_id",
+          "cancellable"
+        ],
+        "properties": {
+          "id": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "identifier": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "additionalProperties": false,
```

Comment:
Additional properties was not included in the docs, but I've added it to all objects within the schema because it's the only way we'll find out about new properties.